openedx/core/djangoapps/course_groups/views.py

```diff
@@ -89,7 +96,26 @@ def _get_cohort_representation(cohort, course):
     }
 
 
-@require_http_methods(("GET", "PUT", "POST"))
+def get_cohorted_discussion_ids(course, cohorted_discussions):
```

Comment:
I think it would make more sense to call this function `get_cohorted_discussions()` and pass it `cohort_settings` so that it can pick the `cohorted_discussions` from it itself.